clusterloader2/cmd/clusterloader.go

```diff
@@ -80,6 +81,7 @@ func validateClusterFlags() *errors.ErrorList {
 func initFlags() {
 	flags.StringVar(&clusterLoaderConfig.ReportDir, "report-dir", "", "Path to the directory where the reports should be saved. Default is empty, which cause reports being written to standard output.")
 	flags.BoolEnvVar(&clusterLoaderConfig.EnablePrometheusServer, "enable-prometheus-server", "ENABLE_PROMETHEUS_SERVER", false, "Whether to set-up the prometheus server in the cluster.")
+	flags.BoolEnvVar(&clusterLoaderConfig.EnableExecPod, "enable-exec-pod", "ENABLE_EXEC_POD", false, "Whether to set-up the exec pod in the cluster.")
```

Comment:
It's no longer a pod - it's a deployment.
But I'm not convinced it should even be called "enable-exec-deployment".

Maybe even more like "enable-exec-service"? [I mean more high-level service meaning, than specific Kubernetes service meaning.]

@mm4tt - thought?

Reply:
Changed to "enable-exec-service".